rpcclient/wallet.go

```diff
@@ -141,6 +141,24 @@ func (c *Client) ListTransactionsCountFrom(account string, count, from int) ([]b
 	return c.ListTransactionsCountFromAsync(account, count, from).Receive()
 }
 
+// ListTransactionsCountFromWatchAsync returns an instance of a type that can be used
+// to get the result of the RPC at some future time by invoking the Receive
+// function on the returned instance.
+//
+// See ListTransactionsCountFrom for the blocking version and more details.
+func (c *Client) ListTransactionsCountFromWatchAsync(account string, count, from int, watch bool) FutureListTransactionsResult {
+	cmd := btcjson.NewListTransactionsCmd(&account, &count, &from, &watch)
+	return c.sendCmd(cmd)
+}
+
+// ListTransactionsCountFrom returns a list of the most recent transactions up
```

Comment:
This doc comment needs to be tweaked, should start with `ListTransactionsCountFromWatch` and mention that it includes watch-only wallet TXs